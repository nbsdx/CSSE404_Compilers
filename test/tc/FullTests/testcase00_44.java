class foo {
	public static void main(String[] args) {
		System.out.println(bar.fib(4));
	}
}

class baz {
	public int fib(int a) {
		return true;
	}
}