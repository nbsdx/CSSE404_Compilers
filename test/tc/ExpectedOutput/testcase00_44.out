Variable bar is not declared.
No method named fib found for class null
In MiniJava, System.out.println only takes an int.  The expression has type null
Actual return type boolean of method fib does not match declared type int
